### PR TITLE
Add special requirement in EIP-1 for mentioning EVM instructions

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
Ping @Arachnid @nicksavers @gcolvin @Souptacular

**Comment by nicksavers** [comment]
This is pretty specific, but sure it seems like the obvious thing. Are there any EIP (Draft or Final) that currently don't meet this requirement?

**Comment by axic** [comment]
There area lot. At point started to fix them and ended up with a few hundred lines of changes, but never finished it. I can push the branch separately, but wouldn't pollute this PR.

File: EIPS/eip-1.md [hunk 1]
```diff
@@ -35,6 +35,13 @@ It is highly recommended that a single EIP contain a single key proposal or new
 
 An EIP must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
 
+### Special requirements for Core EIPs
+
+If a **Core** EIPs changes the EVM (Ethereum Virtual Machine), it should refer to the instructions by their mnemonics and define the opcodes of those mnemonics at least once. A preferred way is the following:
```
Comment (holgerd77) [inline EIPS/eip-1.md]: EIPs -> EIP

Suggestion for some rephrasing: "...mentions or proposes changes to instructions of the EVM"

Comment (axic) [reply EIPS/eip-1.md]: Thanks! Updated.

**Comment by axic** [comment]
@nicksavers pushed here a branch (https://github.com/axic/EIPs/tree/eip1-opcodes-updates) which fixes it for many files (but not all).

**Comment by axic** [comment]
@nicksavers @Arachnid can we merge this?

**Pull request merged**